src/Vasont.Inspire.Models.csproj

```diff
@@ -20,14 +20,14 @@
     <RepositoryType>git</RepositoryType>
     <PackageTags>CCMS</PackageTags>
     <PackageReleaseNotes>Added ReviewerNotificationModel</PackageReleaseNotes>
-    <Version>1.1.99</Version>
+    <Version>1.1.100</Version>
```

Comment:
I'm ok with this. but when do we go to 1.2.0? I think we might instead of going to 1.1.101?

Reply:
perhaps lol